apps/blog/src/api/query-helpers/post.ts

```
@@ -21,11 +25,17 @@ export class PostQueryHelper {
     return {} satisfies Prisma.PostInclude;
   }
 
+  public static adminInclude() {
+    return {
+      ...this.detailInclude(),
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)